src/api/app/assets/javascripts/webui2/request.js

```diff
@@ -106,40 +106,9 @@ function requestShowReview() { // jshint ignore:line
 function requestAddReviewAutocomplete() { // jshint ignore:line
 
   $('#review_type').change(function () {
-    switch ($('#review_type option:selected').attr('value')) {
-      case "user":
-      {
-        $('#review_user_span').show();
-        $('#review_group_span').hide();
-        $('#review_project_span').hide();
-        $('#review_package_span').hide();
-      }
-        break;
-      case "group":
-      {
-        $('#review_user_span').hide();
-        $('#review_group_span').show();
-        $('#review_project_span').hide();
-        $('#review_package_span').hide();
-      }
-        break;
-      case "project":
-      {
-        $('#review_user_span').hide();
-        $('#review_group_span').hide();
-        $('#review_project_span').show();
-        $('#review_package_span').hide();
-      }
-        break;
-      case "package":
-      {
-        $('#review_user_span').hide();
-        $('#review_group_span').hide();
-        $('#review_project_span').show();
-        $('#review_package_span').show();
-      }
-        break;
-    }
+    $('span').addClass('d-none');
```

Comment:
This is going to add the class `d-none` to *all* the `span` elements. Is this really what you want to achieve?

Reply:
nope, i narrowed it down to ```#add-reviewer-modal span```